### ISSUE TITLE
Every time a particular project loads, the parameter to a particular action in a particular action chain is unset

### ISSUE THREAD

**Issue by k98kurz** [FlutterFlow/flutterflow-issues#1396]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

Every time I load a particular project, no matter which browser I use, a particular parameter for a particular custom action in a particular action chain gets unset. I then set it, save, reload, and find it has been unset again.

### Expected Behavior

Saved changes should not be automatically undone when loading a project.

### Steps to Reproduce

1. Replicate this particular project or perhaps make a very complex project with a lot of custom code imported from a team library.
2. Save and load.
3. Get into a vicious cycle of fixing the broken action parameter, saving, reloading, fixing the broken action parameter, saving, reloading...
4. Try restoring a snapshot, saving, and reloading. Same bug occurs.

Idk what else to say about this. It is obviously a Flutter Flow bug -- just watch the video. I was many hours into this project when this started happening, and I doubt very seriously that the FF dev team wants to spend hours replicating a project to see if this particular bug occurs, so just open the project that already exists.

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT5TlPHlx5NOmN9J7arAacZvmjs5JjsjauYzju1ERRQaJuvxEIlze8nRT3FJZte+S3NcDmKZnngK0qHSveDqVsA4ZBCCaYNe/qpAdg7veFutMMWOPJOOfnJTP5hXG26By8CsmA9dKsdfSXws1FOENuutFynZHpy0Om4zQ+SYFIbMon/mFSXsFDhFxmNecCfn

### Context

Trying to load a project that I previously saved.

### Visual documentation

Could not upload the screen capture video directly because the file is too large, so I uploaded it to YouTube.

[Video demonstrating this bug across multiple browsers](https://youtu.be/ddX5TB6sPZ4)

### Environment

```markdown
- FlutterFlow version: v3.1
- Platform: Web
- Browser name and version: Vivaldi 6.1.3035.302 and Chromium 116.0.5845.140
- Operating system and version affected: Ubuntu 22.04
```

**Comment by wenkaifan0720** [comment]
Hi @k98kurz. Looks like the parameter is set properly now in your project?

**Comment by k98kurz** [comment]
@wenkaifan0720 unfortunately, the problem has persisted into the new version of Flutter Flow.

**Comment by wenkaifan0720** [comment]
I checked your project and don't see the error. Could you try clearing the cache of the browser or open the project in incognito mode?

**Comment by k98kurz** [comment]
I cleared the cache and even opened dev tools and disabled the cache in the network tab. Same behavior as demonstrated in the video in both Vivaldi and Chromium. It takes a moment for everything to load before the value unsets and the errors appear in the UI, but it happens every time I open the project.

**Comment by wenkaifan0720** [comment]
https://github.com/FlutterFlow/flutterflow-issues/assets/112435854/415d55bc-e5e0-4246-97c3-fc8502138e19

I still can't reproduce the issue. One thing I noticed is that you are running Ubuntu. I don't have a Ubuntu machine right now. Could you try a Mac or a Windows machine (although I highly doubt it is an OS issue)?

**Comment by k98kurz** [comment]
I do not have a Mac, but I will try Windows. (Bit of a hassle to switch, though, so it'll be a bit before I get to it.)

**Comment by hariprasadms** [comment]
Hi, Any updated on this ? I presume its been resolved to you with latest FF updates. If this is the case let us know I will close the issue. Thanks

**Comment by k98kurz** [comment]
Sorry, I've been preoccupied and have not had a chance to check recently, though the issue persisted the last time I checked. I'll check again tonight and update you.

**Comment by k98kurz** [comment]
On Linux, the behavior changed slightly: the first time after loading the new Flutter Flow version, instead of unsetting the action parameter that decodes the `EdPubKey` value, it unset the action parameter that decodes the `XPubKey` value (same action chain, later action), though the original bugged behavior has returned. I will check it on Windows in a bit.

There is also a perennial project warning "Custom Code: Custom widgets have not been compiled or validated. Click here to check for errors." In the previous Flutter Flow version, it would show up on project load and then disappear after clicking on it once; now, it never goes away, no matter how many times I click it.

**Comment by k98kurz** [comment]
Tested in Windows and encountered the same behavior with Vivaldi. (Vivaldi, Chrome, and Chromium all use the same engine, and I demonstrated that they have the same behavior on Linux, so I'm not going to bother installing, authenticating, and testing on all of them on Windows.)

Note that I test with cache disabled via browser dev tools to ensure I do not have a stale build of FF.

**Comment by wenkaifan0720** [comment]
I still can't reproduce this issue with an Mac. I'll get back to you when I get access to an Windows machine.

**Comment by k98kurz** [comment]
In general, I have found Flutter Flow to be a frustrating experience and a basically useless platform for anything other than a thing, inept client shell on top of server-side logic. I've wasted at least a hundred hours trying to figure out workarounds for bugs and basic missing features; for example, it is impossible to get use the list item index when generating children, so it is impossible to update app state using a child component or page, and it is impossible to get state out of a child component into the outer context to create the update, so components are mostly useless. I have to include a third party library to do basic state management like updating an item in a list. The latest update even removed the ability to change the sizes of columns, so now I can't update or replicate existing UI. Maybe if y'all focused on basic usability rather than all the integrations, this would be worth the $70 I wasted on it.
